### PR TITLE
[DO NOT MERGE] riff-distro system download

### PR DESCRIPTION
**Pull request by glyn**
Fixes https://github.com/projectriff/riff/issues/755

### CODE REVIEW

**Comment by glyn** [comment]
Flagging as DO NOT MERGE since this PR is stacked on another.

File: distro/commands/system.go [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+	command := &cobra.Command{
+		Use:   "download",
+		Short: "Download a riff system.",
+		Long: "Download the kubernetes configuration files for a given riff manifest.\n\n" +
+			"Use the `--output` flag to specify the path of a " +
+			"directory to contain the resultant kubernetes configuration files and rewritten riff manifest." +
+			"The riff manifest is rewritten to refer to the downloaded configuration files.\n",
+		Example: `  riff system download --manifest=path/to/manifest.yaml --output=path/to/output/dir`,
+		PreRunE: func(cmd *cobra.Command, args []string) error {
```
Comment (ericbottard) [inline distro/commands/system.go]: Should be changed to use the `Args` cobra field. Will do myself

File: distro/commands/wiring.go [hunk 2]
```diff
@@ -55,8 +55,14 @@ See https://projectriff.io and https://github.com/knative/docs`,
 		ImagePull(&imageClient),
 	)
 
+	system := System()
```
Comment (ericbottard) [inline distro/commands/wiring.go]: Wondering if "system" is the best word to use. Although this is the `distro` tool, the word "system" here would have a different meaning than in `riff`.

Why not make `download` a level 1 command? This way, users run `riff-distro download`?

**Comment by ericbottard** [comment]
Merged as f8b4bce6 with revised Args validation

**Pull request closed without merging**